valinta-tulos-valintarekisteri-db/src/main/scala/fi/vm/sade/valintatulosservice/valintarekisteri/db/impl/SijoitteluRepositoryImpl.scala

```diff
@@ -353,4 +357,29 @@ trait SijoitteluRepositoryImpl extends SijoitteluRepository with Valintarekister
     }
     exists
   }
+
+  //Poistaa sijoittelun tuloksia yksittäiseltä hakemukselta yksittäisessä hakukohteessa.
```

Comment:
Tässä voisi käyttää javadoc-kommenttia:

```suggestion
   /** Poistaa sijoittelun tuloksia yksittäiseltä hakemukselta yksittäisessä hakukohteessa.
    * ...   
```